src/Symfony/Component/DependencyInjection/Tests/ParameterBag/ContainerBagTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Tests\ParameterBag;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\DependencyInjection\ParameterBag\ContainerBag;
+use Symfony\Component\DependencyInjection\ParameterBag\FrozenParameterBag;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBag;
+
+class ContainerBagTest extends TestCase
+{
+    /** @var ParameterBag */
+    private $parameterBag;
+    /** @var ContainerBag */
+    private $containerBag;
+
+    public function setUp()
+    {
+        $this->containerBag = new ContainerBag(new Container($this->parameterBag = new ParameterBag(array(
```

Comment:
on two lines would be more readable

Reply:
Oh. I love that **you** are saying that, let me change this right now 😛